src/test/java/io/ddavison/conductor/FrameworkTest.java

```diff
@@ -26,6 +26,11 @@ public void testSetText() throws Exception {
     }
 
     @Test
+    public void testSetAndValidateText() throws Exception {
+        setAndValidateText("#setTextField", "set and validate text test");
```

Comment:
in your test, could you add a 

```java
.validateText("set and validate text test");
```

so there is a visible validation in the test?